kahuna/public/js/preview/image.html

```diff
@@ -44,12 +44,15 @@
         <div class="flex-container">
             <ui-labeller-compact class="preview__labeller"
                                  image="ctrl.image"
-                                 disabled="ctrl.selectionMode">
+                                 disabled="ctrl.selectionMode"
+                                 ng:if="!ctrl.inputtingLabel">
             </ui-labeller-compact>
             <span class="flex-spacer"></span>
             <gr-add-label ng:if="!ctrl.selectionMode"
                           image="ctrl.image"
-                          gr-small="true">
+                          gr-small="true"
+                          active="ctrl.inputtingLabel"
+                ng:class="{'gr-add-label--inactive': !ctrl.inputtingLabel}">
```

Comment:
Also for BEM correctness this element should also always have the block class `gr-add-label`, so that the modified `inactive` modifies an actual block. Not a biggie here though.